gradle/libs.versions.toml

```
@@ -12,8 +12,8 @@ spotbugs = "4.7.0"
 snakeyaml = "1.30" # Version set by Bukkit
 
 # Adventure & MiniMessage
-adventure-api = "4.9.3"
-adventure-text-minimessage = "4.1.0-SNAPSHOT"
+adventure-api = "4.10.1"
+adventure-text-minimessage = "4.10.1"
 adventure-platform-bukkit = "4.0.1"
```

```suggestion
adventure-platform-bukkit = "4.0.1"
```
Any specific reason why we don't use 4.1.0?